### PR TITLE
Update CAX DVF PV suffixes

### PR DESCRIPTION
**Pull request by RafaelLyra8**

### CODE REVIEW

File: pyqt-apps/siriushla/bl_ap_imgproc/util.py [hunk 1]
```diff
@@ -57,7 +57,7 @@
     'Image': [
         (2, 0, 1, 3),
         {
-            'Projection': ['image1:ArrayData', 'ImgSizeX-Mon'],
+            'Projection': ['image1:ArrayData', 'image1:ArraySize0_RBV'],
```
Comment (henriquesimoes) [inline pyqt-apps/siriushla/bl_ap_imgproc/util.py:60]: @xresende, @RafaelLyra8 and I discovered that this was causing the image not be properly rendered in GUI, even though the `image1:ArrayData` PV was connected (as we saw during our tests).

What do you think about having this metadata taken from the image1 plugin instead of the high-level IOC?

Comment (xresende) [reply pyqt-apps/siriushla/bl_ap_imgproc/util.py:60]: I think it is cleaner to remove it from the plugin, if i understood it correctly.

Comment (henriquesimoes) [reply pyqt-apps/siriushla/bl_ap_imgproc/util.py:60]: Let me rephrase it. HLA currently displays the image based on the data that comes directly from areaDetector plugin through `image1:ArrayData` PV. However, its shape metadata is taken from the high-level IOC (`ImgSizeX-Mon`). Since during our tests, the high-level IOC was not properly working, we could not see the image (as array data could not be reshaped by the GUI). By using the `image1:ArraySize0_RBV` metadata, we can have the image properly displayed regardless of the status of the high-level IOC.

What do you think of having the GUI relying only in the `image1` plugin for plotting the raw image? (Other image statistics that compose the plot of course can continue existing and being based on imgproc softIOC PVs).

For me, it makes more sense to do so as data is taken directly from there.

Comment (xresende) [reply pyqt-apps/siriushla/bl_ap_imgproc/util.py:60]: I see. I agree with you. We can use `image1` for plotting the image! can you please do this change? or ask @RafaelLyra8 to do it ?

Comment (henriquesimoes) [reply pyqt-apps/siriushla/bl_ap_imgproc/util.py:60]: Good to know you agree. The change in the diff I selected is enough for making everything work the way I described. So, no further changes will be required. ;)

**Review by henriquesimoes — APPROVED** [review]
